rosdep/python.yaml

```
@@ -3107,6 +3107,19 @@ python-opengl:
     xenial: [python-opengl]
     yakkety: [python-opengl]
     zesty: [python-opengl]
+python-openhsi:
```

Please check the [contributing guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-rules)

```suggestion
python-openhsi-pip:
```

If you instead want the python3 version, you should add:

```suggestion
python3-openhsi-pip:
```

(in that case, the key needs to be re-alphabetized)

---

fixed in https://github.com/ros/rosdistro/pull/32533/commits/555f81f0dfecf3b7df493b9162867b36f890f31d